src/main/archetype/core/pom.xml

```diff
@@ -166,6 +172,13 @@ Bundle-DocURL:
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.models.api</artifactId>
         </dependency>
+#else
+        <dependency>
```

Comment:
e.g. package {{org.osgi.framework}} is not part of the sdk (https://osgi.org/javadoc/osgi.core/7.0.0/org/osgi/framework/package-summary.html) and some annotations are neither (https://osgi.org/javadoc/osgi.cmpn/7.0.0/org/osgi/service/component/annotations/package-summary.html). Please either fix the SDK or still add the additional dependencies.

Reply:
AFAICT from looking at https://downloads.experiencecloud.adobe.com/content/maven/public/com/adobe/aem/aem-sdk-api/2020.02.2149.20200205T205013Z-200130/aem-sdk-api-2020.02.2149.20200205T205013Z-200130.pom, these should be included as transitive dependencies.

Reply:
Indeed it is, but I don't understand why osgi.core is a transitive dependency, while all other packages are directly being embedded. I would understand if only the build time annotations would be dependencies, but org.osgi.framework is not only a build time package!

Reply:
does it matter whether we have a transitive dependency or embedded? Eventually we'll move to more transitive depedencies, but this will take some time. Point is, you should only need to include this single dependency. I'll update the core dependency to R7

Reply:
IMHO transitive dependencies are better because:
a) there are source artifacts for at least all open source dependencies available
b) you need to download way less because the transitive dependencies change a lot less frequently

Reply:
yes, I know and as I said, we will move to transitive dependencies. But that's a process and needs time. But the important points is, you just need to include our single aem sdk api as a dependency regardless of transitive dependencies or not

Reply:
understood. It is important then though, that this dependency is always referenced with scope "compile" so that embedded and transitive dependencies behave the same!